assignments/1. hello-world/external/index.html

```diff
@@ -4,7 +4,8 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>Internal</title>
+    <title>External</title>
+    <script type="text/javascript" src="script.js"></script>
```

Comment:
It's a good practice to put `./` infront of the file you are linking. Like `src="./script.js"`

Reply:
Noted